puppet/modules/freight/manifests/install.pp

```diff
@@ -3,7 +3,7 @@
   if $::operatingsystem == 'Debian' {
     apt::source { 'freight':
       location    => 'http://packages.rcrowley.org',
```

Comment:
https://github.com/freight-team/freight#from-a-debian-archive would be used these days on Debian/Ubuntu. ;)